packages/now-static-build/src/utils/gatsby-config.ts

```diff
@@ -0,0 +1,75 @@
+import { writeFile, copyFile, mkdir } from 'fs';
+import { join, dirname } from 'path';
+import { promisify } from 'util';
+
+const writeFilePromise = promisify(writeFile);
+const mkdirPromise = promisify(mkdir);
+const copyFilePromise = promisify(copyFile);
+
+const GATSBY_PLUGIN_NOW_NAME = 'gatsby-plugin-now';
```

Comment:
Below you are using `require.resolve('./gatsby-plugin-now/package.json')` but here you have the name hardcoded?

Is this supposed to represent the package name on npm? Maybe this will work?

```suggestion
const GATSBY_PLUGIN_NOW_NAME = require('./gatsby-plugin-now/package.json').name;
```

Reply:
On second thought, this probably won't work.

So maybe the resolves below should use `GATSBY_PLUGIN_NOW_NAME`?

Reply:
`GATSBY_PLUGIN_NOW_NAME` is representing where the package is copied into the gatsby project of the user (under plugins/GATSBY_PLUGIN_NOW_NAME).

The resolves mentioning `gatsby-plugin-now` are not really related because they are specifying where the original files should be copied **from** and **not to** (they are related to the build script!).

Reply:
Ok I understand now. I was thinking that "inject" would mean `yarn install gatsby-plugin-now` but I see now its just copying the source files from `gatsby-plugin-now`.